pandas/tools/merge.py

```diff
@@ -1436,13 +1438,198 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
     copy : boolean, default True
         If False, do not copy data unnecessarily
 
-    Notes
-    -----
-    The keys, levels, and names arguments are all optional
-
     Returns
     -------
     concatenated : type of objects
+
+    Notes
+    -----
+    The keys, levels, and names arguments are all optional.
+
+    A walkthrough of how this method fits in with other tools for combining
+    panda objects can be found `here </pandas-docs/stable/merging.html>`_.
+
+    See Also
+    --------
+    Series.append
+    DataFrame.append
+    DataFrame.join
+    DataFrame.merge
+    Panel.join
+
+    Examples
+    --------
+    Combine two ``Series``.
+
+    >>> s1 = pd.Series(['a', 'b'])
+    >>> s2 = pd.Series(['c', 'd'])
+    >>> pd.concat([s1, s2])
+    0    a
+    1    b
+    0    c
+    1    d
+    dtype: object
+
+    Clear the existing index and reset it in the result
+    by setting the ``ignore_index`` option to ``True``.
+
+    >>> s1 = pd.Series(['a', 'b'])
+    >>> s2 = pd.Series(['c', 'd'])
+    >>> pd.concat([s1, s2], ignore_index=True)
+    0    a
+    1    b
+    2    c
+    3    d
+    dtype: object
+
+    Add a hierarchical index at the outermost level of
+    the data with the ``keys`` option.
+
+    >>> s1 = pd.Series(['a', 'b', 'c'])
+    >>> s2 = pd.Series(['c', 'd', 'e'])
+    >>> pd.concat([s1, s2], keys=['s1', 's2',])
+    s1  0    a
+        1    b
+        2    c
+    s2  0    c
+        1    d
+        2    e
+    dtype: object
+
+    Label the index keys you create with the ``names`` option.
+
+    >>> s1 = pd.Series(['a', 'b', 'c'])
+    >>> s2 = pd.Series(['c', 'd', 'e'])
+    >>> pd.concat(
+    ...     [s1, s2],
+    ...     keys=['s1', 's2',],
+    ...     names=['Series name', 'Row ID']
+    ... )
```

Comment:
No, no problem. We don't have this documented clearly what is expected here (and are not always that consistent in existing docs)